tests/caasp/oci_register.pm

```diff
@@ -10,9 +10,10 @@
 # Summary: Type in registration information
 # Maintainer: Martin Kravec <mkravec@suse.com>
 
+use base 'installbasetest';
+use y2_logs_helper;
```

Comment:
I'm not sure if that is right, as `y2_logs_helper` is new `y2logsstep`, but with your change postfail_hooks, etc. will be called from `installbasetest`. `y2logsstep` mainly contains things for the YaST installer. As per proposal we should have gotten `y2_install_base_test` instead and have it as a parent class instead of `y2logsstep`.

Reply:
Point is, we don't want to change behavior of the code, we only change the structure. Previously, `post_fail_hook` would be executed from `lib/y2logsstep.pm`, with this change, it will be executed from `installbasetest`. So it's change in the behavior. From the proposal we should have `y2_installbasetest` (not sure about naming), which should include relevant parts from current `y2logsstep`. And common parts can be moved somewhere else, like to the helper you have created.
Hope it's more clear now.

Reply:
clear enough. let me take care of it